api/src/pcapi/scripts/offerers_with_one_venue_are_permanent/main.py

```diff
@@ -0,0 +1,69 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions):
+
+https://github.com/pass-culture/pass-culture-main/blob/pcharlet/pc-32809-script-offerer-with-one-venue-is-permanent/api/src/pcapi/scripts/offerers_with_one_venue_are_permanent/main.py
+
+"""
+
+import argparse
+import csv
+import logging
+import os
+
+from flask_sqlalchemy import BaseQuery
+import sqlalchemy as sqla
+
+from pcapi.app import app
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def extract_venues_info_to_csv(query: BaseQuery) -> None:
+    venues_query_for_csv = query.with_entities(
+        offerers_models.Venue.id, offerers_models.VenueContact.email, offerers_models.Venue.hasOffers
+    ).join(offerers_models.VenueContact, offerers_models.VenueContact.venueId == offerers_models.Venue.id)
```

Comment:
`VenueContact` n'existe que s'il y a au moins une info de contact remplie, c'est nullable, donc tu vas perdre des `Venue`s en route.
```suggestion
    ).outerjoin(offerers_models.VenueContact, offerers_models.VenueContact.venueId == offerers_models.Venue.id)
```

Mais à quoi cette collecte d'email est-elle destinée ? Selon ma compréhension, `VenueContact.email` est l'adresse email communiquée sur l'app jeunes pour contacter l'acteur culturel. Mais c'est l'adresse `Venue.bookingEmail` qui est utilisée pour des notifications par le pass Culture.

Reply:
`bookingEmail` obsolète ? C'est cette adresse qui porte les attributs de la `Venue` dans Brevo, c'est aussi à cette adresse que sont envoyés pas mal d'emails.

Reply:
je parlais de l'autre champ `VenueContact.email`

Reply:
tout à fait

Reply:
Du coup si le `VenueContact` est obsolète, je vais utiliser `bookingEmail` ce qui va régler le soucis du join/outerjoin

Reply:
L'objectif est de lister les venues concernées par cette mise à jour et pouvoir les contacter pour leur dire qu'ils ont désormais une page partenaire.